crates/rmf_site_editor/src/widgets/tasks/mod.rs

```diff
@@ -337,7 +301,7 @@ fn show_task(
     } else {
         Color32::default()
     };
-    let mut edit = false;
+    let in_edit_mode = edit_task.0.is_some_and(|e| e == task_entity);
```

Comment:
It's not exactly here (a few lines below) but I'm a bit conflicted on using the task's `Entity` value for its display name, has there been a thought on having a `NameInSite` component for tasks? We could auto-generate it with a random ID the way we do in the RMF stack. Or if we want to go for IDs maybe just the `SiteID`? `Entity` values can be pretty random, I noticed that creating a new task in my test (screenshots in main review comment) created `Task 1078`, and creating a new one immediately after resulted in `Task 1081`, I would argue the very high + not even contiguous number is not too informative.